jetson/percep/artag_detector.hpp

```diff
@@ -31,4 +31,4 @@ class TagDetector {
     double getAngle(float xPixel, float wPixel);     
     //if AR tag found, updates distance, bearing, and id                              
     void updateDetectedTagInfo(rover_msgs::Target *arTags, pair<Tag, Tag> &tagPair, Mat &depth_img, Mat &src); 
-};
```

Comment:
It's good practice to have a line at the end of the file, so please add this back in.

Reply:
Added the line back